scripts/test-automation/alcor_services.ini

```diff
@@ -0,0 +1,26 @@
+[services]
+ignite = {"img_name":"ignite", "name":"ignite-11","path":"/lib/ignite.Dockerfile"}
```

Comment:
what is the difference between img_name and name?

Reply:
This will go away as we discussed this morning that name of ignite service and image will be same (unlike before when image was ignite-11 and name was ignite).